packages/client/src/internal/conversions/modernToLegacyWhereClause.test.ts

```diff
@@ -26,10 +26,18 @@ describe(modernToLegacyWhereClause, () => {
   describe("api namespaces", () => {
     describe("interfaces", () => {
       it("properly converts shortname to fqApiName", () => {
-        const T: ObjectOrInterfaceDefinition = {
+        const T = {
           type: "interface",
           apiName: "a.Foo",
-        };
+          __DefinitionMetadata: {
```

Comment:
Had to add all this info because our old whereClause type would just default to `NumberFilters` for anything where the types didn't match the rest of our branching logic